yarn.lock

```diff
@@ -2264,10 +2244,10 @@
     resolve-from "^5.0.0"
     store2 "^2.12.0"
 
-"@symphony-ui/uitoolkit-styles@1.5.0-rc.3":
-  version "1.5.0-rc.3"
-  resolved "https://registry.yarnpkg.com/@symphony-ui/uitoolkit-styles/-/uitoolkit-styles-1.5.0-rc.3.tgz#91c662c40a18e7e1d790ccdc5602df631d9dfb83"
-  integrity sha512-wPHGLBphvT07MyHyp8jnNa4OO0LLW0EzUyZbBIXs8Ennus27Z8G21df23w0sma4UC3v4m9o+EVCU+LwLZKrJ+w==
+"@symphony-ui/uitoolkit-styles@1.5.0-rc.4":
```

Comment:
1.5.0 was already release, The RC should be for 1.6.0

Reply:
ooh no